### DISCUSSION TITLE
Question: keybind to send unicode like "\u0002c" that mimics "<ctrl-b>c" in tmux

### DISCUSSION THREAD

**Discussion by zuxfoucault** [ghostty-org/ghostty#3253] [Q&A] [answered]
Thanks for the great work! I am migrating from Alacritty and translating my Alacritty config like:

"""
[[keyboard.bindings]]
chars = "\u0002c"
key = "T"
mods = "Command"

[[keyboard.bindings]]
chars = "\u00021"
key = "Key1"
mods = "Option"

[[keyboard.bindings]]
chars = "\u00022"
key = "Key2"
mods = "Option"

[[keyboard.bindings]]
chars = "\u00023"
key = "Key3"
mods = "Option"
"""

It doesn't work with the following config in ghostty:
"""
keybind = cmd+t=text:\u0002c
keybind = opt+1=text:\u00021
keybind = opt+2=text:\u00022
keybind = opt+3=text:\u00023
"""

Any suggestions would be appreciated!

**Comment by zuxfoucault** [comment, marked as answer]
I figured it out! I can't directly insert the unicode but need to do as https://x-team.com/magazine/inserting-unicode-characters-in-vim

Then it works.

**Comment by vlanx** [comment]
For posteriority also

I mapped `cmd+a` to `Ctrl+B` for the tmux prefix like so:
```
keybind = super+a=text:\x02
```

Just needed to get the `Ctrl+B`, which i got from [this table](https://www3.rocketsoftware.com/bluezone/help/v42/en/bzadmin/APPENDIX_C/ASCII_Character_Set.htm)
